frontend/cypress/integration/featureFiles/app_details_multicluster.feature

```diff
@@ -0,0 +1,66 @@
+@multi-cluster
+@to-be-implemented
```

Comment:
Can we use `@skip` here rather than adding a new tag filter? https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/tags.md#smart-tagging

Reply:
Sorry. I had overlooked the fact that Gherkin has a built-in tag for this.